assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -27,18 +27,24 @@ import invariant from 'invariant';
 import API from 'googlesitekit-api';
 import Data from 'googlesitekit-data';
 import { STORE_NAME } from './constants';
+import { STORE_NAME as CORE_USER } from '../../datastore/user/constants';
 import { createFetchStore } from '../../data/create-fetch-store';
 
-const { createRegistrySelector } = Data;
+const { createRegistrySelector, createRegistryControl } = Data;
 
 // Actions.
 const REFETCH_AUTHENICATION = 'REFETCH_AUTHENICATION';
 
 const fetchGetModulesStore = createFetchStore( {
 	baseName: 'getModules',
-	controlCallback: () => {
+	argsToParams: ( { useCache = true } = {} ) => {
+		return {
+			options: { useCache },
+		};
+	},
+	controlCallback: ( { options: { useCache } } ) => {
 		return API.get( 'core', 'modules', 'list', null, {
-			useCache: false,
+			useCache,
```

Comment:
This change shouldn't be necessary as we no longer need `useCache` to be variable here.